### PR TITLE
Use Travis CI for continuous integration builds per-PR

### PR DESCRIPTION
**Pull request by jeffmay**
@DSASanFrancisco/portal-members 

Sorry, @scudmissile, I didn't like how Circle CI could not be disabled for branches that don't have a .circle.yml file so I dug into the differences between Circle CI and Travis CI. I think Travis better supports the open-source community, provides what we need with a simple interface, and has some advantages, like S3 caching, that might be useful for UI builds. I'm open to switching to Circle CI if there are any big advantages in build time or configurability, but for now, I'd say let's go with the company that best serves the community.

This would fulfill the same purpose as https://github.com/DSASanFrancisco/membership_api/pull/30

### CODE REVIEW

File: ci.env [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# This file is used for CI builds
+
+# If enabled, you'll have to authenticate with Auth0
+USE_AUTH=FALSE
+
+# If USE_AUTH is set to false, then set the authenticated user to this user
+# This should match a user in the database
+NO_AUTH_EMAIL=joe.schmoe@example.com
```
Comment (dbolson) [inline ci.env]: Should this be a real email address?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+before_install:
+
+  # install docker-compose
+  - sudo -i curl -L https://github.com/docker/compose/releases/download/$DOCKER_COMPOSE_VERSION/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+  - sudo chmod +x /usr/local/bin/docker-compose
+
+  # check that docker-compose is now running the latest version (or at least the
+  # one we specified). This is not to be confused with the version we printed
+  # before doing the update.
+  - docker-compose --version
```
Comment (dbolson) [inline .travis.yml]: Is this just informational?

Comment (jeffmay) [reply .travis.yml]: Yea, for debugging. I can probably remove it now.

File: ci.env (hunk 1 above)
Comment (jeffmay) [reply ci.env]: Probably not. I've disabled the email feature for this anyway.

File: .travis.yml (hunk 2 above)
Comment (jeffmay) [reply .travis.yml]: Ok, removed and rebased.

**Pull request merged**